mindsdb/api/executor/utilities/sql.py

```diff
@@ -130,13 +130,16 @@ def adapt_query(node, is_table, **kwargs):
                 node.parts = [node.parts[-1]]
                 return node
         if isinstance(node, Function):
+            # These functions also exist in DuckDB, therefore, the results need to be replaced.
+            fnc_results = {
+                "database": session.database if session else None,
+                "version": "8.0.17",
```

Comment:
What is it for? Was some test broken without it?

Reply:
Yes. Theis test started to fail: https://github.com/mindsdb/mindsdb/blob/e6dea5e69a09af883d89a2cf78a0cc194ebae83f/tests/unit/executor/test_base_queires.py#L601

I believe the reason is because DuckDB also implements both `VERSION()` and `CURRENT_SCHEMA()` functions, similar to `DATABASE()`.